meteor/client/ui/SegmentTimeline/SourceLayerItem.tsx

```diff
@@ -356,6 +411,58 @@ export const SourceLayerItem = withTranslation()(
 			}
 		}
 
+		// TODO(Performance): use ResizeObserver to avoid style recalculations
+		// checkElementWidth = () => {
+		// 	if (this.state.itemElement && this._forceSizingRecheck) {
+		// 		this._forceSizingRecheck = false
+		// 		const width = getElementWidth(this.state.itemElement) || 0
+		// 		if (this.state.elementWidth !== width) {
+		// 			this.setState({
+		// 				elementWidth: width
+		// 			})
+		// 		}
+		// 	}
+		// }
+
+		private measureElement = () => {
```

Comment:
I don't think this is needed anymore, since `this.state.elementWidth` is no longer used.